docs/AdminTools.md

```diff
@@ -400,7 +400,7 @@ It provisions a new cluster in Pulsar. It also requires Pulsar super-user privil
 ###### CLI
 
 ```
-$ pulsar-admin clusters create --url http://my-cluster.org.com:8080/ cl1
+$ pulsar-admin clusters create --url http://my-cluster.org.com:8080/ --broker-url pulsar://my-cluster.org.com:6650/ cl1
```

Comment:
Changes in documentation should be postponed to when we release 1.16, otherwise people would get errors in this case